pages/common/gt.md

```diff
@@ -0,0 +1,32 @@
+# gt
+
+> Create and manage sequences of dependent code changes (stacks) for Git and GitHub.
+> More information: <https://docs.graphite.dev>.
+
+- Authenticate the CLI with Graphite's API:
+
+`gt auth --token {{graphite_cli_auth_token}}`
+
+- Initialise `gt` for the repository in the current directory:
+
+`gt repo init`
+
+- Create a new branch stacked on top of the current branch and commit staged changes:
```

Comment:
```suggestion
- Create the new branch stacked on top of the current branch and commit staged changes:
```

Reply:
On `a` vs `the` vs dropping article entirely, I think we should go with with the same style as `tldr git-foo` pages (e.g. [here](https://github.com/tldr-pages/tldr/blob/main/pages/common/git-commit.md)) where they either go with `a` or _no article_.
At the time the command is written "the" resulting branch does not exist yet and could never exist (e.g. if the command fails), similarly you could run the command multiple times and create multiple branches, so can see why they made that language choice.

What are your thoughts?

Reply:
`a` should be used here. cc @CleanMachine1

Reply:
Yes, agreed. Using `the` here doesn't fit the context.